mk/external.mk

```diff
@@ -23,18 +23,26 @@ QUAKE_DATA_VERIFY = echo "$(strip $$($(T)_DATA_SHA1))  $$@" | $(SHA1SUM) -c
 # Timidity software synthesizer configuration for SDL2_mixer
 TIMIDITY_DATA_URL = http://www.libsdl.org/projects/mixer/timidity/timidity.tar.gz
 TIMIDITY_DATA = $(OUT)/timidity
-TIMIDITY_DATA_SHA1 = cdd30736508d26968222a6414f3beabc3b7a0725
+TIMIDITY_DATA_SHA1 = cf6217a5d824b717ec4a07e15e6c129a4657ca25
 TIMIDITY_DATA_EXTRACT = tar -xf $(notdir $($(T)_DATA_URL)) -C $(OUT)
-TIMIDITY_TMP_FILE = /tmp/timidity_sha1.txt
-TIMIDITY_DATA_VERIFY = echo "$(TIMIDITY_DATA_SHA1)" > $(TIMIDITY_TMP_FILE) | find $(TIMIDITY_DATA) -type f -print0 | sort -z | xargs -0 shasum | shasum | cut -f 1 -d ' '
+TIMIDITY_TMP_FILE1 = /tmp/timidity_sha1.txt
+TIMIDITY_TMP_FILE2 = /tmp/timidity_sha2.txt
+TIMIDITY_DATA_VERIFY = echo "$(TIMIDITY_DATA_SHA1)" > $(TIMIDITY_TMP_FILE1) \
```

Comment:
Can you make it modular? That is, provide calls for zip and (tar.)gz respectively?